docs/setup/manual-setup.md

```diff
@@ -24,29 +27,28 @@
 
 3. Enter the directory and type `bin/setup`, which will install npm dependencies and create configs.
 
-4. Modify `config.json` or configure CodiMD through environment variables which will
-   overwrite the configs, see docs [here](https://github.com/codimd/server/blob/master/docs/configuration.md).
+4. Create `config.json` for example by copying `config.json.example` or configure HedgeDoc through environment variables which will overwrite the configs, see docs [here](https://github.com/hedgedoc/server/blob/master/docs/configuration.md).
 
 5. Build front-end bundle by `yarn run build` (use `yarn run dev` if you are in development)
 
-6. Modify the file named `.sequelizerc`, change the value of the variable `url` with your db connection string. For example:
-   - `postgres://username:password@localhost:5432/codimd`
-   - `mysql://username:password@localhost:3306/codimd`
+6. Create the file named `.sequelizerc` for example by copying it from `.sequelizerc.example` and change the value of the variable `url` with your db connection string. For example:
+   - `postgres://username:password@localhost:5432/hedgedoc`
+   - `mysql://username:password@localhost:3306/hedgedoc`
    - `sqlite://:memory:`
 
-7. It is recommended to start your server manually once: `npm start --production`, this way it's easier to see warnings or errors that might occur (leave out `--production` for development).
+7. It is recommended to start your server manually once: `yarn start --production`, this way it's easier to see warnings or errors that might occur (leave out `--production` for development).
```

Comment:
What about #239?

Reply:
As far as I understand both https://classic.yarnpkg.com/en/docs/cli/run/ and https://docs.npmjs.com/cli/v6/commands/npm-run-script pass all arguments to the script they call.
But I did not find any place where we actually *do* something with that argument.
So `NODE_ENV=production yarn start` might be the best way.